SYCL/InvokeSimd/Spec/simd_size/Inputs/common.hpp

```diff
@@ -6,15 +6,6 @@
 #include <iostream>
 #include <type_traits>
 
-/* Subgroup size attribute is optional
- * In case it is absent compiler decides what subgroup size to use
- */
-#ifdef IMPL_SUBGROUP
```

Comment:
Sorry why are we changing this? Do we not want to test it without an explicit subgroup size?

Reply:
It turned out that cases where subgroup size does not match size of invoke_simd return type or uniform argument type are not allowed.
At this point main purpose of most simd_size cases is to make sure that the compiler reports an error when an invalid configuration is used.
All tests that expect to compile and execute successfully now contain only one case where the sizes are equal.
This configuration is widely used in other InvokeSimd tests, that have implicit subgroup size versions.

Reply:
thanks!